styles/base/_form.scss

```diff
@@ -137,9 +145,16 @@ label {
     min-height: $form-size;
     vertical-align: middle;
     padding: 0 8px;
-    background: $brand-gray-lt;
-    border: 1px solid $brand-gray;
+    border: 1px solid $brand-black;
     border-radius: $border-radius-base;
+    background-color: $brand-white;
+    color: $brand-black;
+  }
+  
+  > .addon.-primary-link {
+    border-color: $brand-sidebar-lt;
+    background-color: $brand-sidebar-lt;
+    color: $brand-white;
```

Comment:
Isn't this a little bit of confusing? co'z the variant name is `-primary` but the color use is `$brand-sidebar-lt` ?